packages/ui/src/proposals/components/ProposalDetails/ProposalDetails.tsx

```diff
@@ -72,6 +75,24 @@ export const ProposalDetails = ({ proposalDetails }: Props) => {
   const detailsRenderStructure = useMemo(() => getDetailsRenderStructure(proposalDetails), [proposalDetails])
 
   const additionalDetails = useMemo(() => {
+    if (gracePeriod) {
+      return [
+        {
+          renderType: 'BlockDurationStatistics',
+          title: 'Gracing Period',
+          value: gracePeriod,
+        },
+      ] as unknown as RenderNode[]
+    }
+    if (exactExecutionBlock) {
+      return [
+        {
+          renderType: 'BlockDurationStatistics',
```

Comment:
Finally this https://github.com/Joystream/pioneer/pull/4478#discussion_r1327571671 still stands. `exactExecutionBlock` is a date not a duration so it shouldn't be shown has a `BlockDurationStatistics` but has something which looks like [BlockTime](https://pioneer-2-storybook-joystream.vercel.app/?path=/story/common-blocktime--block-time-view).

Reply:
BTW I just added `exactExecutionBlock` in the stories so you can test it there. Just set it to a non zero value in the controls.